spec/service_consumers/pact_helper.rb

```diff
@@ -204,4 +204,34 @@
       FactoryGirl.create(:version, target: arabic_draft, number: 1)
     end
   end
+
+  provider_state "the content item bed722e6-db68-43e5-9079-063f623335a7 is at version 3" do
+    set_up do
+      DatabaseCleaner.clean_with :truncation
+
+      draft = FactoryGirl.create(:draft_content_item, content_id: "bed722e6-db68-43e5-9079-063f623335a7")
+      FactoryGirl.create(:version, target: draft, number: 3)
+
+      stub_default_url_arbiter_responses
+      stub_request(:put, /content-store/)
```

Comment:
I guess what I'm saying is they don't feel right here at all since the stubs are related to the behaviour which is related to the request more than the setup, so I'd rather get this merged and then talk about how we can do it better.